src/core/types.ts

```diff
@@ -1,36 +1,74 @@
 import { CustomTranslations, Locales } from '../language/types';
+import { PaymentAmount, PaymentMethods, PaymentMethodOptions } from '../types';
+import { AnalyticsOptions } from './Analytics/Analytics';
+import { PaymentMethodsResponseObject } from './ProcessResponse/PaymentMethodsResponse/types';
+import { RiskModuleOptions } from './RiskModule/RiskModule';
 
 export interface CoreOptions {
     /**
-     * Use test. When you're ready to accept live payments, change the value to one of our {@link https://docs.adyen.com/checkout/drop-in-web#testing-your-integration }live environments}.
+     * Use test. When you're ready to accept live payments, change the value to one of our {@link https://docs.adyen.com/checkout/drop-in-web#testing-your-integration | live environments}.
      */
     environment?: 'test' | 'live' | 'live-us' | 'live-au' | string;
```

Comment:
Shouldn't it be an enum if the values are fixed?

Reply:
That'd make a lot of sense indeed, although at this point environment can be more things besides those values. We'll keep it in mind for a future PR, thanks! :)